kedro_vertexai/cli.py

```diff
@@ -99,16 +99,21 @@ def run_once(
     """Deploy pipeline as a single run within given experiment
     Config can be specified in kubeflow.yml as well."""
     context_helper = ctx.obj["context_helper"]
-    config = context_helper.config.run_config
+    config: RunConfig = context_helper.config.run_config
     client: VertexAIPipelinesClient = context_helper.vertexai_client
 
-    client.run_once(
+    run = client.run_once(
         pipeline=pipeline,
         image=image if image else config.image,
         image_pull_policy=config.image_pull_policy,
         parameters=format_params(params),
     )
 
+    click.echo(
+        f"Intermediate data datasets will be stored in{os.linesep}"
+        f"gs://{config.root.strip('/')}/{KEDRO_VERTEXAI_BLOB_TEMP_DIR_NAME}/{run['displayName']}/*.bin"
```

Comment:
`run['displayName']` is used in the message, but files land in `run.run_id` subdir - are these same?

Reply:
Actually, display name is used here (https://github.com/getindata/kedro-vertexai/pull/63/files#diff-efca27e2c6efab2c456a964844cf6ce99c04171ab0640a66ed358bd18a633a66R49), but indeed - the parameter in the `KedroVertexAIRunnerDataset` is named `run_id`, which is misleading. I will rename it to `unique_id`.